execution/executor-benchmark/src/pipeline.rs

```diff
@@ -63,12 +76,14 @@ where
             Self {
                 join_handles,
                 phantom: PhantomData,
+                start_commit: start_commit_1,
             },
             block_sender,
         )
     }
 
     pub fn join(self) {
+        self.start_commit.store(true, Ordering::Relaxed);
```

Comment:
If we just want to benchmark execution, can we just not turn off commit altogether?

Reply:
The speculative state tree update results will be kept in mem any way.